cluster-autoscaler/hack/update-mods.sh

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
```

Comment:
We should check if required go version in CA `go.mod` is the same as one in `k8s`.

Reply:
Done